docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -1,11 +1,12 @@
-### Style guide for Flutter repo
+# Style guide for Flutter repo
 
 ## Summary
 
-Optimize for readability. Write detailed documentation.
-Make error messages useful.
-Never use timeouts or timers.
-Avoid `is`, `print`, `part of`, `extension` and `_`.
+- Optimize for readability.
+- Write detailed documentation.
```

Comment:
This would be OK as a list, but it's not wrong as it is — it's a stylistic difference.

The prose paragraph of five short pithy sentences has a bit of an effect of hammering the points at the reader. The list has a much blunter and heavier hammering effect. I'm not sure that's an improvement here.

Reply:
I comprehend a list with greater ease and speed than a paragraph. 

> ## Summary
> 
> - Optimize for readability.
> - Write detailed documentation.
> - Make error messages useful.
> - Never use timeouts or timers.
> - Avoid `is`, `print`, `part of`, `extension` and `_`.

vs

> ## Summary
> 
> Optimize for readability. Write detailed documentation. Make error messages useful. Never use timeouts or timers. Avoid `is`, `print`, `part of`, `extension` and `_`.

I'm not under the impression that people relish documentation and read it thoroughly, so it may be useful to the Flutter project to employ a blunt and easy-to-parse summary. 

That being said, I see merit in the paragraph, too. I will revert. If you change your mind and like the list better, let me know.